boxtree/fmm.py

```diff
@@ -51,7 +51,7 @@ def drive_fmm(traversal, expansion_wrangler, src_weights, timing_data=None):
     :arg traversal: A :class:`boxtree.traversal.FMMTraversalInfo` instance.
     :arg expansion_wrangler: An object exhibiting the
         :class:`ExpansionWranglerInterface`.
-    :arg src_weights: Source 'density/weights/charges'.
+    :arg src_weights: A sequence of source 'density/weights/charges'.
```

Comment:
```suggestion
    :arg src_weight_vecs: A sequence of source 'density/weights/charges'.
```
(I.e. please rename this everywhere, to make the cardinality clear.)